03.asynchronous/callback_non_error.js

```diff
@@ -0,0 +1,29 @@
+import sqlite3 from "sqlite3";
+
+const db = new sqlite3.Database(":memory:", () => {
+  db.run(
+    "CREATE TABLE books(id INTEGER PRIMARY KEY AUTOINCREMENT NOT NULL, title VARCHAR(20) UNIQUE NOT NULL)",
+    () => {
+      db.run(
+        "INSERT INTO books (title) VALUES (?)",
+        "JavaScript本格入門",
+        function () {
+          console.log(`ID:${this.lastID}`);
```

Comment:
スタイル的に空白を入れたほうが良さそうです。

```suggestion
          console.log(`ID:${this.lastID}`);
```

Reply:
すみません、Suggested changeには変更がないように見えるのですが、修正すべき箇所を教えていただけますでしょうか。🙇

Reply:
おっと、すみません。コロンの後ろです。

```suggestion
          console.log(`ID: ${this.lastID}`);
```

Reply:
修正箇所、承知いたしました。
合わせて修正しました。